receptai.api/Controllers/SubfoodditUserController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using receptai.api.Interfaces;
+using System.Threading.Tasks;
+
+namespace receptai.api.Controllers;
+
+[Route("api/subfooddit-users")]
+[ApiController]
+public class SubfoodditUserController : ControllerBase
+{
+    private readonly ISubfoodditRepository _subfoodditRepository;
+
+    public SubfoodditUserController(ISubfoodditRepository subfoodditRepository)
+    {
+        _subfoodditRepository = subfoodditRepository;
+    }
+
+    [HttpPost("{userId:int}")]
+    public async Task<IActionResult> AddUser(int subfoodditId, int userId)
+    {
+        var success = await _subfoodditRepository.AddUserToSubfooddit(subfoodditId, userId);
+        if (!success)
+        {
+            return NotFound("Subfooddit or user not found.");
+        }
+
+        return Ok();
+    }
```

Comment:
Only user himself should be able to be added to subfoodit.

Use [Authorize] and get user id from User.GetId();

Reply:
Fixed.